taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py

```diff
@@ -69,6 +70,8 @@ def run(self):
                             break
                         job = self.orchestrator.jobs_to_run.get(block=True, timeout=0.1)
                     self._execute_job(job)
+                else:
+                    time.sleep(0.1)  # We need to sleep to avoid busy waiting.
```

Comment:
💭 Maybe this value should be configurable via env variable/configuration.

Reply:
Not sure that would bring any value

Reply:
I think it might. Here we arbitrarily wait for 0.1 second. Why not 0.01 ? Or 1 second?

Reply:
The number of workers is configurable. I see no reason for wait time in dispatcher loop not to be configurable as well. 
It should just come with reasonable defaults (and `0.1` looks a reasonable value).

Reply:
I think this `sleep()` is really technical ie it allows to give back control to the system and the value is not really important here: it should be the minimum possible.

Reply:
I agree with Gregoire, though then it makes the issue a bit more complicated as too much option isn't necessarily a good thing. If we decide to work on it, maybe a different PR would be better too

Reply:
I totally agree ! This is not blocking for me and could eventually be addressed later if there is a need.